### DISCUSSION TITLE
tabularray not working in Overleaf

### DISCUSSION THREAD

**Discussion by zoepierrat** [lvjr/tabularray#390] [Q&A]
I cannot for the life of me figure out what I'm doing wrong. I get this error message

\l__tblr_template_caption-tag_default_tl ...ename 
                                                  \hspace {0.25em}\thetable 
l.72 \end
         {longtblr}
The control sequence at the end of the top line
of your error message was never \def'ed. If you have
misspelled it (e.g., `\hobx'), type `I' and the correct
spelling (e.g., `I\hbox'). Otherwise just continue,
and I'll forget about whatever was undefined.

! Undefined control sequence.
\l__tblr_template_caption-tag_default_tl ...ename 
                                                  \hspace {0.25em}\thetable 
l.72 \end
         {longtblr}
The control sequence at the end of the top line
of your error message was never \def'ed. If you have
misspelled it (e.g., `\hobx'), type `I' and the correct
spelling (e.g., `I\hbox'). Otherwise just continue,
and I'll forget about whatever was undefined.

**Comment by muzimuzhi** [comment]
any reproducible example?

**Reply by zoepierrat** [reply]
I figured it out - I needed to change the version of TexLive I was using to 2021 - not sure why but this seemed to fix it
